openvdb_cmd/vdb_tool/CMakeLists.txt

```diff
@@ -0,0 +1,159 @@
+cmake_minimum_required(VERSION 3.10)
+
+# CMP0091 allows for MSVC ABI targetting via CMAKE_MSVC_RUNTIME_LIBRARY
+# from CMake 3.15 and above. Must come before project().
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
+project(vdb_tool LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 14)
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE "Release" CACHE STRING
+      "Choose the type of build, supported options are: Debug Release."
+      FORCE)
+endif()
+
+list(APPEND CMAKE_MODULE_PATH ${OPENVDB_CMAKE_PATH})
+
+# Pseudo library to manage dependencies
+add_library(vdb_tool_common INTERFACE)
+
+# Optional components
+option(BUILD_TEST "Build unit tests" OFF)
+
+option(USE_NANO "Compile with NanoVDB support" ON)
```

Comment:
I can't see any attempt to get this to work with AX. Have you considered how this might clash with the math operations you've added?

Reply:
no you I understand your concern but let's discuss. You're definitely right that I haven't attempted to add AX support yet but I'll need your help do define exactly what that means